tests/scadnano_tests.py

```diff
@@ -671,19 +671,26 @@ def test_circular_auto_staple_hex(self) -> None:
         # To help with debugging, uncomment these lines to write out the
         # scadnano file
         #
-        # design.write_scadnano_file(directory=self.output_path,
-        #                            filename=f'{file_name}.{sc.default_scadnano_file_extension}')
+        design.write_scadnano_file(directory=self.output_path,
+                                   filename=f'{file_name}.{sc.default_scadnano_file_extension}')
 
     def test_paranemic_crossover(self) -> None:
-        file_name = "test_paranemic_crossover"
+        file_name = "test_crossover_to_same_helix"
         design = sc.Design.from_cadnano_v2(directory=self.input_path,
                                            filename=file_name + ".json")
-        self.assertEqual(4, len(design.helices))
+        self.assertEqual(1, len(design.helices))
         # To help with debugging, uncomment these lines to write out the
         # scadnano file
         #
-        # design.write_scadnano_file(directory=self.output_path,
-        #                            filename=f'{file_name}.{sc.default_scadnano_file_extension}')
+        design.write_scadnano_file(directory=self.output_path,
+                                   filename=f'{file_name}.{sc.default_scadnano_file_extension}')
+    
+    def test_same_helix_crossover(self) -> None:
+        file_name = "test_paranemic_crossover"

```

Comment:
Is this a typo? Seems like file name should be "test_same_helix_crossover" instead of "test_paranemic_crossover"

Reply:
I'd call it `test_same_helix_crossover`. A paranemic crossover is one that connects domains going the same direction on both helices, which is technically true here, but this is more a "fake" crossover that doesn't really represent the DNA geometry. (Unlike paranemic crossover, which is an actual DNA motif.)